homeassistant/components/harmony/remote.py

```diff
@@ -178,6 +141,39 @@ def __init__(self, name, host, port, activity, out_path, delay_secs):
         self._delay_secs = delay_secs
         self._available = False
 
+    @property
+    def delay_secs(self):
+        """Delay seconds between sending commands."""
+        return self._delay_secs
+
+    @delay_secs.setter
```

Comment:
Remove the setter. Please implement our dispatch helper to change entity features. We should avoid interacting with entities from the outside directly and never from outside the platform directly.

Reply:
done